baton/modules/giphy/giphy.go

```diff
@@ -55,16 +56,16 @@ func (g Giphy) RunCommand(cmd string, body interface{}, resp chan<- interface{})
 	resp <- out
 	return nil
 }
-func (g Giphy) makeURL(path string, v url.Values) (url.URL, error){
-  baseURL, _ := url.Parse("http://api.giphy.com/v1/")
-  v.Add("api_key",g.ApiKey)
-   
-  rel, err := url.Parse(path)
-  if err != nil{
-    return url.URL{}, err
-  }
-  
-  u := baseURL.ResolveReference(rel)
-  u.RawQuery = v.Encode()
-  return *u, nil
-}
\ No newline at end of file
+func (g Giphy) makeURL(path string, v url.Values) (url.URL, error) {
+	baseURL, _ := url.Parse("http://api.giphy.com/v1/")
+	v.Add("api_key", g.ApiKey)
+
+	rel, err := url.Parse(path)
+	if err != nil {
+		return url.URL{}, err
+	}
+
+	u := baseURL.ResolveReference(rel)
+	u.RawQuery = v.Encode()
+	return *u, nil
+}
```

Comment:
In the future can we always make sure to `gofmt` files before committing? Not doing so makes diffs like these more difficult to read.